packages/block-editor/src/components/block-list/style.scss

```diff
@@ -1090,18 +1090,16 @@
 
 	.components-toolbar {
 		border: none;
-		line-height: 1;
 		font-family: $default-font;
```

Comment:
Should `font-family` be in the mixin by default? The consumer can still choose to override the `font-family` if they have a valid exception.

Reply:
Seems like we could get rid of font-family in the mixin.